woocommerce/woocommerce-gutenberg-products-block
# Store Customization > Add default image to Just Arrived pattern

## What

Adds default image and height to the Just Arrived pattern. Note that we could not use the same image as displayed in the related issue due to licensing.

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/11091

## Why



All patterns need a default image which may (or may not) be replaced by AI, the pattern design also had a fixed height.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new post and add the pattern "Just Arrived Full Hero"
2. Check that it looks like the below screenshots on both desktop and mobile viewports both in the editor and on the frontend.


## Screenshots or screencast



| Desktop | Mobile |
| ------ | ----- |
| ![Screenshot 2023-10-04 at 14 21 14](https://github.com/woocommerce/woocommerce-blocks/assets/8639742/321440db-4be3-4691-af48-427edbe60488) | ![Screenshot 2023-10-04 at 14 21 28](https://github.com/woocommerce/woocommerce-blocks/assets/8639742/90b55b88-e7da-4555-b25e-480ad4474a22) |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add default image and fixed height to the Just Arrived Full Hero pattern.
